paddle/fluid/operators/math/blas.h

```diff
@@ -112,6 +112,15 @@ class Blas {
 
   template <typename T>
   void GEMM_FREE(T* data) const;
+
+#if !defined(PADDLE_WITH_CUDA)
```

Comment:
Sometimes `#if !defined(PADDLE_WITH_CUDA)` and sometimes `#ifdef PADDLE_WITH_MKLML` are used for `MatMulWithHead`.

Reply:
You are right. I will make the macro consistent for multiple head.

Reply:
This `#if` is still inconsistent with other `#if`s for code with `MatMulWithHead()`.

Reply:
there is already a `#ifdef PADDLE_WITH_MKLML` outside the definition of `MatMulWithHead`, like below:
```C
#ifdef PADDLE_WITH_MKLML
...

#if !defined(PADDLE_WITH_CUDA)
  template <typename T>
  void MatMulWithHead(const framework::Tensor& mat_a,
                      const MatDescriptor& dim_a,
                      const framework::Tensor& mat_b,
                      const MatDescriptor& dim_b, T alpha, int head_number,
                      framework::Tensor* mat_out, T beta) const;
#endif
#endif
```
so they are consistent with other code:)

Reply:
OK, that's fine then. Thanks!